# Pybliographer German Translation
# Copyright (C) 2005 Free Software Foundation, Inc.
# Christian Meyer <linux@chrisime.de>, 2000.
# Frank Arnold <frank@scirocco-5v-turbo.de>, 2005.
# Mario Blättermann <mariobl@gnome.org>, 2008.
#
msgid ""
msgstr ""
"Project-Id-Version: pybliographer-1.0.x\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2009-01-02 00:19+0100\n"
"PO-Revision-Date: 2009-01-02 00:26+0100\n"
"Last-Translator: Mario Blättermann <mariobl@gnome.org>\n"
"Language-Team: German <gnome-de@gnome.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: ../Pyblio/Autoload.py:78
#, python-format
msgid "warning: can't import %s: %s"
msgstr "Warnung: %s kann nicht importiert werden: %s"

#: ../Pyblio/Config.py:52
#, python-format
msgid "value of `%s' should be of type %s"
msgstr "Wert von »%s« sollte vom Typ %s sein"

#: ../Pyblio/Config.py:212
msgid "String"
msgstr "String"

#: ../Pyblio/Config.py:221
msgid "Boolean"
msgstr "Boolean"

#: ../Pyblio/Config.py:240
msgid "Integer"
msgstr "Integer"

#: ../Pyblio/Config.py:242
#, python-format
msgid "Integer under %d"
msgstr "Integer unter %d"

#: ../Pyblio/Config.py:244
#, python-format
msgid "Integer over %d"
msgstr "Integer über %d"

#: ../Pyblio/Config.py:246
#, python-format
msgid "Integer between %d and %d"
msgstr "Integer zwischen %d und %d"

#: ../Pyblio/Config.py:258
#, python-format
msgid "Element in `%s'"
msgstr "Element in »%s«"

#: ../Pyblio/Config.py:278
#, python-format
msgid "Tuple (%s)"
msgstr "Sequenz (Tupel) (%s)"

#: ../Pyblio/Config.py:299
#, python-format
msgid "List (%s)"
msgstr "Liste (%s)"

#: ../Pyblio/Config.py:324
#, python-format
msgid "Dictionary (%s, %s)"
msgstr "Indizierte Liste (Dictionary) (%s, %s)"

#: ../Pyblio/Config.py:344
#, python-format
msgid "warning: could not restore setting %r to %r: %s"
msgstr "Warnung: Einstellung %r kann nicht auf %r wiederhergestellt werden: %s"

#: ../Pyblio/Fields.py:261
msgid "Illegal year value"
msgstr "Falscher Wert für Jahr"

#: ../Pyblio/Fields.py:265
msgid "Illegal month value"
msgstr "Falscher Wert für Monat"

#: ../Pyblio/Fields.py:269
msgid "Illegal day value"
msgstr "Falscher Wert für Tag"

#: ../Pyblio/Fields.py:510
#, python-format
msgid "in %s"
msgstr "in %s"

#: ../Pyblio/Format/BibTeX.py:323 ../Pyblio/Format/BibTeX.py:371
#, python-format
msgid "key `%s' is malformed"
msgstr "Syntax des Schlüssels »%s« ist falsch"

#: ../Pyblio/Format/BibTeX.py:340 ../Pyblio/Format/BibTeX.py:359
#, python-format
msgid "%s:%d: key `%s' already defined"
msgstr "%s:%d: Schlüssel »%s« schon definiert"

#: ../Pyblio/GnomeUI/Config.py:147 ../Pyblio/GnomeUI/Fields.py:417
msgid ""
"Some changes require to restart Pybliographic\n"
"to be correctly taken into account"
msgstr ""
"Einige Änderungen erfordern den Neustart von »Pybliographer«\n"
"um korrekt übernommen zu werden"

#: ../Pyblio/GnomeUI/Config.py:429
msgid "Add"
msgstr "Hinzufügen"

#: ../Pyblio/GnomeUI/Config.py:432
msgid "Update"
msgstr "Auffrischen"

#: ../Pyblio/GnomeUI/Config.py:435 ../Pyblio/GnomeUI/Config.py:549
msgid "Remove"
msgstr "Entfernen"

#: ../Pyblio/GnomeUI/Config.py:546
msgid "Set"
msgstr "Festlegen"

#: ../Pyblio/GnomeUI/Config.py:561
msgid "Key:"
msgstr "Schlüssel:"

#: ../Pyblio/GnomeUI/Config.py:563
msgid "Value:"
msgstr "Wert:"

#. id     stock            label         accel   tooltip   callback
#: ../Pyblio/GnomeUI/Document.py:145
msgid "_File"
msgstr "_Datei"

#: ../Pyblio/GnomeUI/Document.py:146 ../Pyblio/GnomeUI/Document.py:195
msgid "_Edit"
msgstr "_Bearbeiten"

#: ../Pyblio/GnomeUI/Document.py:147
msgid "_View"
msgstr "_Ansicht"

#: ../Pyblio/GnomeUI/Document.py:148
msgid "_Cite"
msgstr "_Zitat"

#: ../Pyblio/GnomeUI/Document.py:149
msgid "_Settings"
msgstr "_Einstellungen"

#: ../Pyblio/GnomeUI/Document.py:150
msgid "_Help"
msgstr "_Hilfe"

#: ../Pyblio/GnomeUI/Document.py:151
msgid "Recent documents"
msgstr "Zuletzt geöffnete Dokumente"

#: ../Pyblio/GnomeUI/Document.py:154
msgid "Open a file"
msgstr "Datei öffnen"

#: ../Pyblio/GnomeUI/Document.py:155
msgid "Open _Location"
msgstr "_Ort öffnen"

#: ../Pyblio/GnomeUI/Document.py:156
msgid "Save the current file"
msgstr "Aktuelle Datei speichern"

#: ../Pyblio/GnomeUI/Document.py:161
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:8
msgid "Merge With..."
msgstr "Zusammenführen mit…"

#: ../Pyblio/GnomeUI/Document.py:162
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:7
msgid "Medline Query..."
msgstr "Medline-Abfrage…"

#: ../Pyblio/GnomeUI/Document.py:170
msgid "Add a new entry"
msgstr "Neuen Eintrag hinzufügen"

#: ../Pyblio/GnomeUI/Document.py:174
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:12
msgid "S_ort..."
msgstr "S_ortieren…"

#: ../Pyblio/GnomeUI/Document.py:175
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:2
msgid "Cite..."
msgstr "Zitat…"

#: ../Pyblio/GnomeUI/Document.py:175
msgid "Cite key(s)"
msgstr "Schlüssel zitieren"

#: ../Pyblio/GnomeUI/Document.py:176
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:6
msgid "Format..."
msgstr "Format…"

#: ../Pyblio/GnomeUI/Document.py:178
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:4
msgid "Fields..."
msgstr "Felder…"

#: ../Pyblio/GnomeUI/Document.py:180
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:5
msgid "Forget all changes"
msgstr "Alle Änderungen verwerfen"

#: ../Pyblio/GnomeUI/Document.py:183
msgid "_Resource"
msgstr "_Ressource"

#: ../Pyblio/GnomeUI/Document.py:194
msgid "_About"
msgstr "_Info"

#: ../Pyblio/GnomeUI/Document.py:242
msgid "Quick search"
msgstr "Schnellsuche"

#: ../Pyblio/GnomeUI/Document.py:316
msgid "Unnamed bibliographic database"
msgstr "Unbenannte bibliographische Datenbank"

#: ../Pyblio/GnomeUI/Document.py:399
#, python-format
msgid ""
"can't open file `%s' for writing:\n"
"%s"
msgstr ""
"Datei »%s« kann nicht zum Schreiben geöffnet werden:\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:420
#, python-format
msgid ""
"Error while parsing `%s':\n"
"%s"
msgstr ""
"Fehler beim Parsen »%s«:\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:436
msgid "New database"
msgstr "Neue Datenbank"

#: ../Pyblio/GnomeUI/Document.py:444 ../Pyblio/GnomeUI/Document.py:448
msgid "[no entry]"
msgstr "[kein Eintrag]"

#: ../Pyblio/GnomeUI/Document.py:445
msgid "[1 entry]"
msgstr "[1 Eintrag]"

#: ../Pyblio/GnomeUI/Document.py:446
#, python-format
msgid "[%d entries]"
msgstr "[%d Einträge]"

#: ../Pyblio/GnomeUI/Document.py:449
#, python-format
msgid "[%d/1 entry]"
msgstr "[%d/1 Eintrag]"

#: ../Pyblio/GnomeUI/Document.py:450
#, python-format
msgid "[%d/%d entries]"
msgstr "[%d/%d Einträge]"

#: ../Pyblio/GnomeUI/Document.py:455
msgid "[modified]"
msgstr "[geändert]"

#: ../Pyblio/GnomeUI/Document.py:458
#, python-format
msgid "view limited to: %s"
msgstr "Ansicht begrenzt auf: %s"

#: ../Pyblio/GnomeUI/Document.py:471
msgid ""
"The database has been modified.\n"
"Discard changes?"
msgstr ""
"Diese Datenbank wurde geändert.\n"
"Änderungen verwerfen?"

#: ../Pyblio/GnomeUI/Document.py:475
msgid ""
"The database has been modified.\n"
"Save changes?"
msgstr ""
"Diese Datenbank wurde geändert.\n"
"Änderungen speichern?"

#: ../Pyblio/GnomeUI/Document.py:492
msgid ""
"The database has been externally modified.\n"
"Overwrite changes ?"
msgstr ""
"Diese Datenbank wurde extern verändert.\n"
"Änderungen überschreiben?"

#. no result.
#: ../Pyblio/GnomeUI/Document.py:521
msgid "Your query returned no result"
msgstr "Die Abfrage lieferte kein Ergebnis"

#. get a new file name
#: ../Pyblio/GnomeUI/Document.py:531
msgid "Merge file"
msgstr "Datei zusammenführen"

#: ../Pyblio/GnomeUI/Document.py:543 ../Pyblio/GnomeUI/Document.py:629
msgid "Open error"
msgstr "Fehler beim Öffnen"

#: ../Pyblio/GnomeUI/Document.py:567
msgid "Merge status"
msgstr "Status der Zusammenführung"

#. get a new file name
#: ../Pyblio/GnomeUI/Document.py:578
msgid "Open file"
msgstr "Datei öffnen"

#: ../Pyblio/GnomeUI/Document.py:616
msgid ""
"An autosave file was found which is newer than the original file.\n"
"Do you want to restore it?"
msgstr ""
"Es wurde eine automatisch gespeicherte Datei gefunden, die aktueller als die "
"Originaldatei ist.\n"
"Möchten Sie diese Datei wiederherstellen?"

#: ../Pyblio/GnomeUI/Document.py:650 ../Pyblio/GnomeUI/Document.py:791
#: ../Pyblio/GnomeUI/Document.py:851
#, python-format
msgid ""
"Unable to remove autosave file `%s':\n"
"%s"
msgstr ""
"Löschen der automatisch gespeicherten Datei »%s« fehlgeschlagen:\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:701
#, python-format
msgid ""
"Error during autosaving:\n"
"%s"
msgstr ""
"Fehler während des automatischen Speicherns:\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:729
#, python-format
msgid ""
"Unable to save `%s':\n"
"%s"
msgstr ""
"»%s« kann nicht gespeichert werden:\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:737
msgid ""
"An internal error occured during saving\n"
"Try to Save As..."
msgstr ""
"Ein interner Fehler trat während des Speicherns auf\n"
"Probieren Sie »Speichern unter…«"

#: ../Pyblio/GnomeUI/Document.py:752
msgid "Save As..."
msgstr "Speichern unter…"

#: ../Pyblio/GnomeUI/Document.py:758
#, python-format
msgid ""
"The file `%s' already exists.\n"
"Overwrite it ?"
msgstr ""
"Die Datei »%s« existiert bereits.\n"
"Überschreiben?"

#: ../Pyblio/GnomeUI/Document.py:765
#, python-format
msgid ""
"During opening:\n"
"%s"
msgstr ""
"Während des Öffnens:\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:803
msgid "Reopen error"
msgstr "Fehler beim erneuten Öffnen"

#: ../Pyblio/GnomeUI/Document.py:876
#, python-format
msgid ""
"An entry called `%s' already exists.\n"
"Rename and add it anyway ?"
msgstr ""
"Ein Eintrag mit dem Namen »%s« existiert bereits.\n"
"Trotzdem umbenennen und hinzufügen ?"

#: ../Pyblio/GnomeUI/Document.py:913
msgid "Really remove all the entries ?"
msgstr "Wirklich alle Einträge entfernen ?"

#: ../Pyblio/GnomeUI/Document.py:933
msgid "Create new entry"
msgstr "Neuen Eintrag erstellen"

#: ../Pyblio/GnomeUI/Document.py:947
#, python-format
msgid "Really edit %d entries ?"
msgstr "Wirklich %d Einträge bearbeiten ?"

#: ../Pyblio/GnomeUI/Document.py:993
#, python-format
msgid "Remove all the %d entries ?"
msgstr "Alle %d Einträge entfernen ?"

#: ../Pyblio/GnomeUI/Document.py:995
#, python-format
msgid "Remove entry `%s' ?"
msgstr "Eintrag »%s« entfernen ?"

#: ../Pyblio/GnomeUI/Document.py:1014
msgid ""
"your search text must contain\n"
"latin-1 characters only"
msgstr ""
"Ihre Suche darf nur\n"
"latin-1-Zeichen enthalten"

#: ../Pyblio/GnomeUI/Document.py:1093 ../Pyblio/GnomeUI/Document.py:1101
#, python-format
msgid ""
"Can't connect to LyX:\n"
"%s"
msgstr ""
"Es kann keine Verbindung zu LyX hergestellt werden:\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:1216
#, python-format
msgid ""
"Can't display documentation:\n"
"%s"
msgstr ""
"Konnte Dokumentation nicht anzeigen:\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:1224
msgid "This program is copyrighted under the GNU GPL"
msgstr "Dieses Programm steht unter der GNU GPL"

#: ../Pyblio/GnomeUI/Document.py:1225
msgid "GNOME interface to the Pybliographer system."
msgstr "GNOME-Schnittstelle zum Pybliographer-System."

#: ../Pyblio/GnomeUI/Document.py:1236
msgid "GNOME Translation Team"
msgstr "GNOME-Übersetzungsteam"

#: ../Pyblio/GnomeUI/Document.py:1241
msgid "Pybliographer Home Page"
msgstr "Pybliographer-Homepage"

#: ../Pyblio/GnomeUI/Editor.py:35
msgid "Last Name, First Name"
msgstr "Name, Vorname"

#: ../Pyblio/GnomeUI/Editor.py:36 ../Pyblio/GnomeUI/Editor.py:1292
#: ../Pyblio/GnomeUI/Fields.py:39 ../Pyblio/GnomeUI/glade/config1.glade.h:14
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:19
msgid "Text"
msgstr "Text"

#: ../Pyblio/GnomeUI/Editor.py:133 ../Pyblio/GnomeUI/Editor.py:1003
msgid "Error in native string parsing"
msgstr "Fehler beim »Stringparsen«"

#: ../Pyblio/GnomeUI/Editor.py:312
msgid "Day"
msgstr "Tag"

#: ../Pyblio/GnomeUI/Editor.py:321
msgid "Month"
msgstr "Monat"

#: ../Pyblio/GnomeUI/Editor.py:330
msgid "Year"
msgstr "Jahr"

#: ../Pyblio/GnomeUI/Editor.py:355
msgid "Invalid day field in date"
msgstr "Ungültiger Tag in Datum"

#: ../Pyblio/GnomeUI/Editor.py:363
msgid "Invalid month field in date"
msgstr "Ungültiger Monat in Datum"

#: ../Pyblio/GnomeUI/Editor.py:371
msgid "Invalid year field in date"
msgstr "Ungültiges Jahr in Datum"

#. A delete button
#: ../Pyblio/GnomeUI/Editor.py:419 ../Pyblio/GnomeUI/Search.py:86
msgid "Delete"
msgstr "Löschen"

#: ../Pyblio/GnomeUI/Editor.py:430 ../Pyblio/GnomeUI/Editor.py:443
#: ../Pyblio/GnomeUI/Editor.py:446
msgid "[ Drop an Entry here ]"
msgstr "[ Eintrag hierher ziehen ]"

#: ../Pyblio/GnomeUI/Editor.py:502
msgid "Browse..."
msgstr "Durchsuchen…"

#: ../Pyblio/GnomeUI/Editor.py:525
msgid "Select file"
msgstr "Datei auswählen"

#: ../Pyblio/GnomeUI/Editor.py:549 ../Pyblio/GnomeUI/Fields.py:236
#: ../Pyblio/GnomeUI/glade/config1.glade.h:8
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:6
msgid "Entry type"
msgstr "Eintragstyp"

#: ../Pyblio/GnomeUI/Editor.py:551
msgid "Key"
msgstr "Schlüssel"

#: ../Pyblio/GnomeUI/Editor.py:600
msgid "Create Field"
msgstr "Feld erstellen"

#. navigation buttons
#: ../Pyblio/GnomeUI/Editor.py:605
msgid "Back"
msgstr "Zurück"

#: ../Pyblio/GnomeUI/Editor.py:608
msgid "Next"
msgstr "Vor"

#: ../Pyblio/GnomeUI/Editor.py:728 ../Pyblio/GnomeUI/Fields.py:451
msgid "Mandatory"
msgstr "Erforderlich"

#: ../Pyblio/GnomeUI/Editor.py:728 ../Pyblio/GnomeUI/Fields.py:452
msgid "Optional"
msgstr "Optional"

#: ../Pyblio/GnomeUI/Editor.py:728 ../Pyblio/GnomeUI/Editor.py:1207
msgid "Notes"
msgstr "Notizen"

#: ../Pyblio/GnomeUI/Editor.py:728
msgid "Extra"
msgstr "Zusatz"

#: ../Pyblio/GnomeUI/Editor.py:909
msgid "Invalid key format"
msgstr "Ungültiges Schlüsselformat"

#: ../Pyblio/GnomeUI/Editor.py:917
#, python-format
msgid "Key `%s' already exists"
msgstr "Schlüssel »%s« ist bereits vorhanden"

#: ../Pyblio/GnomeUI/Editor.py:934
#, python-format
msgid "The `%s' field contains a non Latin-1 symbol"
msgstr ""
"Das »%s«-Feld enthält ein Zeichen, dass nicht im Latin-1 Zeichensatz enthalten "
"ist"

#: ../Pyblio/GnomeUI/Editor.py:996
msgid "Your text contains non Latin-1 symbols"
msgstr ""
"Der Text enthält Zeichen, die nicht im Latin-1 Zeichensatz enthalten sind."

#: ../Pyblio/GnomeUI/Editor.py:1023
msgid "Edit entry"
msgstr "Eintrag bearbeiten"

#: ../Pyblio/GnomeUI/Editor.py:1037 ../Pyblio/GnomeUI/Editor.py:1102
msgid "Native Editing"
msgstr "Bearbeitung von Native"

#: ../Pyblio/GnomeUI/Editor.py:1125
msgid "Standard Editing"
msgstr "Standardbearbeitung"

#: ../Pyblio/GnomeUI/Editor.py:1229
msgid "mandatory"
msgstr "erforderlich"

#: ../Pyblio/GnomeUI/Editor.py:1295
msgid "Enter text here"
msgstr "Text hier eingeben"

#: ../Pyblio/GnomeUI/Editor.py:1347
msgid "New Annotation Name"
msgstr "Neuer Anmerkungsname"

#: ../Pyblio/GnomeUI/Editor.py:1353
msgid "Name of the new annotation:"
msgstr "Name der neuen Anmerkung"

#: ../Pyblio/GnomeUI/Entry.py:114
#, python-format
msgid ""
"Cannot open URL:\n"
"%s"
msgstr ""
"URL konnte nicht geöffnet werden:\n"
"%s"

#: ../Pyblio/GnomeUI/Fields.py:38
msgid "Authors"
msgstr "Autoren"

#: ../Pyblio/GnomeUI/Fields.py:40
msgid "Long Text"
msgstr "Langer Text"

#: ../Pyblio/GnomeUI/Fields.py:41 ../Pyblio/GnomeUI/glade/config1.glade.h:16
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:21
msgid "URL"
msgstr "URL"

#: ../Pyblio/GnomeUI/Fields.py:42
msgid "Reference"
msgstr "Referenz"

#: ../Pyblio/GnomeUI/Fields.py:43
msgid "Date"
msgstr "Datum"

#: ../Pyblio/GnomeUI/Fields.py:124 ../Pyblio/GnomeUI/glade/config1.glade.h:11
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:10
msgid "Name"
msgstr "Name"

#: ../Pyblio/GnomeUI/Fields.py:127 ../Pyblio/GnomeUI/glade/config1.glade.h:15
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:20
msgid "Type"
msgstr "Typ"

#: ../Pyblio/GnomeUI/Fields.py:300
msgid "Available"
msgstr "Verfügbar"

#: ../Pyblio/GnomeUI/Fields.py:310
msgid "Associated"
msgstr "Assoziiert"

#: ../Pyblio/GnomeUI/Fields.py:321
msgid "Please, select an entry type from previous page."
msgstr "Bitte einen Eintragstyp aus der vorhergehenden Seite wählen"

#: ../Pyblio/GnomeUI/Fields.py:332
#, python-format
msgid "Fields associated with <b>%s</b> entry type"
msgstr "Mit Eintragstyp <b>%s</b> assoziierte Felder"

#: ../Pyblio/GnomeUI/FileSelector.py:39
msgid "File"
msgstr "Datei"

#: ../Pyblio/GnomeUI/FileSelector.py:76
#: ../Pyblio/GnomeUI/glade/openurl.glade.h:2
msgid "Bibliography type:"
msgstr "Bibliographietyp:"

#: ../Pyblio/GnomeUI/FileSelector.py:93 ../Pyblio/GnomeUI/OpenURL.py:61
msgid " - According to file suffix - "
msgstr " - Der Dateiendung entsprechend - "

#. The root of the search tree is the full database
#: ../Pyblio/GnomeUI/Search.py:71
msgid "Full database"
msgstr "Gesamte Datenbank"

#: ../Pyblio/GnomeUI/Search.py:142
msgid "internal error during evaluation"
msgstr "Interner Fehler während der Evaluierung"

#: ../Pyblio/GnomeUI/Search.py:196
#, python-format
msgid ""
"while compiling %s\n"
"error: %s"
msgstr ""
"Beim Compilieren von »%s«\n"
"Fehler: %s"

#. These are additional search fields
#: ../Pyblio/GnomeUI/Sort.py:97
msgid "[Entry Type]"
msgstr "[Eintragstyp]"

#: ../Pyblio/GnomeUI/Sort.py:98
msgid "[Key Value]"
msgstr "[Schlüsselvariable]"

#: ../Pyblio/GnomeUI/Utils.py:193
msgid "The following errors occured:\n"
msgstr "Die folgenden Fehler traten auf:\n"

#: ../Pyblio/GnomeUI/__init__.py:45
#, python-format
msgid "This is Pybliographic %s [Python %s, Gtk %s, PyGTK %s]"
msgstr "Dies ist Pybliographic %s [Python %s, Gtk %s, PyGTK %s]"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:1
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:1
#: ../Pyblio/GnomeUI/glade/medline.glade.h:1
#: ../Pyblio/GnomeUI/glade/openurl.glade.h:1
#: ../Pyblio/GnomeUI/glade/search.glade.h:1
msgid "*"
msgstr "*"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:2
msgid "<b>Entry type</b>"
msgstr "<b>Eintragstyp</b>"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:3
msgid "A"
msgstr "A"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:4
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:2
msgid "Association"
msgstr "Assoziation"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:5
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:3
msgid "Datum"
msgstr "Datum"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:6
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:4
msgid "Editor"
msgstr "Editor"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:7
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:5
msgid "Entries"
msgstr "Einträge"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:9
msgid "FIELDS DEFINITION"
msgstr "FELD DEFINITION"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:10
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:8
msgid "Fields "
msgstr "Felder "

#: ../Pyblio/GnomeUI/glade/config1.glade.h:12
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:11
msgid "Name "
msgstr "Name "

#: ../Pyblio/GnomeUI/glade/config1.glade.h:13
msgid "Pybliographer Configuration"
msgstr "Pybliographer Einstellungen"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:17
msgid "standard fields"
msgstr "Vorgabefelder"

#: ../Pyblio/GnomeUI/glade/fields1.glade.h:7
msgid "Entry types and field names configuration"
msgstr "Eintragstypen- und Feldnameneinstellungen"

#: ../Pyblio/GnomeUI/glade/fields1.glade.h:9
msgid "Information for <b>Article</b> entry type"
msgstr "Informationen zum <b>Artikel</b>-Eintragstyp"

#: ../Pyblio/GnomeUI/glade/fields1.glade.h:12
msgid ""
"Select an entry type and specify \n"
"its associated (mandatory and\n"
"optional) fields on the next page.\n"
"\n"
"\n"
"\n"
msgstr ""
"Bitte einen Eintragstyp wählen und seine\n"
"assoziierten (erforderlichen und optionalen)\n"
"Felder auf der nächsten Seite festlegen.\n"
"\n"
"\n"
"\n"

#: ../Pyblio/GnomeUI/glade/format.glade.h:1
msgid "Bibliography style:"
msgstr "Bibliographiestil:"

#: ../Pyblio/GnomeUI/glade/format.glade.h:2
msgid "Format entries"
msgstr "Einträge formatieren"

#: ../Pyblio/GnomeUI/glade/format.glade.h:3
msgid "Output file:"
msgstr "Ausgabedatei:"

#: ../Pyblio/GnomeUI/glade/format.glade.h:4
msgid "Output format:"
msgstr "Ausgabeformat:"

#: ../Pyblio/GnomeUI/glade/format.glade.h:5
msgid "Save Formatted as"
msgstr "Formatiert speichern unter"

#: ../Pyblio/GnomeUI/glade/format.glade.h:6
msgid "Style Selection"
msgstr "Stilauswahl"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:2
msgid "Enter your Medline Query"
msgstr "Medline-Abfrage erstellen"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:3
msgid "From:"
msgstr "Von:"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:4
msgid "Limited to:"
msgstr "Begrenzt auf:"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:5
msgid ""
"Maximum number\n"
"of results:"
msgstr ""
"Maximale Anzahl\n"
"der Ergebnisse:"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:7
msgid "Medline Query"
msgstr "Medline-Abfrage"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:8
msgid ""
"Only items \n"
"ahead of print"
msgstr ""
"Nur Exemplare \n"
"vor Druck"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:10
msgid ""
"Only items \n"
"with abstracts"
msgstr ""
"Nur Exemplare \n"
"mit Zusammenfassung"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:12
msgid ""
"Search \n"
"PubMed for:"
msgstr ""
"Durchsuche \n"
"PubMed nach:"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:14
msgid ""
"Starting listing at\n"
"result number:"
msgstr ""
"Auflistung beginnen \n"
"bei Resultat Nummer:"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:16
msgid "To:"
msgstr "Bis:"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:17
msgid ""
"Use the format YYYY/MM/DD\n"
"Month and day are optional."
msgstr ""
"Format JJJJ/MM/TT verwenden\n"
"Monat und Tag sind optional."

#: ../Pyblio/GnomeUI/glade/openurl.glade.h:3
msgid "Enter the location (URI) of the file you would like to open:"
msgstr "Geben Sie den Ort (URI) der zu öffnenden Datei ein:"

#: ../Pyblio/GnomeUI/glade/openurl.glade.h:4
msgid "Open Location"
msgstr "Ort öffnen"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:1
msgid "Cite"
msgstr "Zitat"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:3
msgid "Contents"
msgstr "Inhalt"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:9
msgid "Open File"
msgstr "Datei öffnen"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:10
msgid "Previous Documents"
msgstr "Vorhergehende Dokumente"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:11
msgid "Pybliographer"
msgstr "Pybliographer"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:13
msgid "Save File"
msgstr "Datei speichern"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:14
msgid "Settings"
msgstr "Einstellungen"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:15
msgid "_Add..."
msgstr "Hinzu_fügen…"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:16
msgid "_Delete..."
msgstr "_Löschen…"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:17
msgid "_Edit..."
msgstr "_Bearbeiten…"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:18
msgid "_New"
msgstr "_Neu"

#: ../Pyblio/GnomeUI/glade/search.glade.h:2
msgid "Close"
msgstr "Schließen"

#: ../Pyblio/GnomeUI/glade/search.glade.h:3
msgid "Expert Search"
msgstr "Erweiterte Suche"

#: ../Pyblio/GnomeUI/glade/search.glade.h:4
msgid "Field"
msgstr "Feld"

#: ../Pyblio/GnomeUI/glade/search.glade.h:5
msgid "Pattern"
msgstr "Muster"

#: ../Pyblio/GnomeUI/glade/search.glade.h:6
msgid "Query"
msgstr "Abfrage"

#: ../Pyblio/GnomeUI/glade/search.glade.h:7
msgid "Search"
msgstr "Suche"

#: ../Pyblio/GnomeUI/glade/search.glade.h:8
msgid "Simple Search"
msgstr "Einfache Suche"

#: ../Pyblio/GnomeUI/glade/sort.glade.h:1
msgid "Select sort criterions"
msgstr "Sortierungskriterien auswählen"

#: ../Pyblio/GnomeUI/glade/sort.glade.h:2
msgid "Set as default"
msgstr "Als Vorgabe setzen"

#: ../Pyblio/LyX.py:37
#, python-format
msgid "no input pipe `%s'"
msgstr "keine Eingabe-Pipe »%s«"

#: ../Pyblio/LyX.py:43
#, python-format
msgid "no output pipe `%s'"
msgstr "keine Ausgabe-Pipe »%s«"

# wie kann man "opener" richtig(!!) übersetzen?
#: ../Pyblio/Open.py:96
#, python-format
msgid "method `%s' provides no opener"
msgstr "Methode »%s« stellt keinen Öffner bereit"

#: ../Pyblio/Open.py:104 ../Pyblio/Open.py:151
#, python-format
msgid "File `%s' does not exist"
msgstr "Datei »%s« existiert nicht"

#: ../Pyblio/Open.py:115 ../Pyblio/Open.py:162
#, python-format
msgid "don't know how to open `%s'"
msgstr "Wie soll »%s« geöffnet werden?"

#: ../Pyblio/Open.py:143
#, python-format
msgid "method `%s' provides no iterator"
msgstr "Methode »%s« unterstützt keinen Iterator"

#: ../Pyblio/Resource.py:77
#, python-format
msgid ""
"Warning: This URL is marked as Invalid or Approximate: %s\n"
"Continue?"
msgstr ""
"Warnung: Diese Adresse ist als »ungültig« oder »annähernd« markiert: %s\n"
"Fortsetzen?"

#: ../Pyblio/Resource.py:81
msgid "Determining Mime Type ... "
msgstr "MIME-Typ wird bestimmt…"

#: ../Pyblio/Resource.py:86
#, python-format
msgid "Cannot determine mime type for item %s "
msgstr "Der MIME-Typ für das Objekt »%s« konnte nicht bestimmt werden"

#: ../Pyblio/Resource.py:87
#, python-format
msgid ""
"URL in question is: %s\n"
"You should check the url or path given for errors.\n"
"Details: %s"
msgstr ""
"Die betroffene Adresse lautet: %s\n"
"Überprüfen Sie diese Adresse oder den gegebenen Pfad auf Fehler.\n"
"Nähere Informationen: %s"

#: ../Pyblio/Resource.py:98
msgid "Accessing resource ..."
msgstr "Zugriff auf die Ressource wird ausgeführt…"

#: ../Pyblio/Resource.py:116
#, python-format
msgid "IOError for item %s: cannot uncompress resource."
msgstr ""
"Ein-/Ausgabefehler für das Objekt %s: Entpacken der Ressource fehlgeschlagen."

#: ../Pyblio/Resource.py:117
#, python-format
msgid ""
"URL: %s\n"
"Details: %s"
msgstr ""
"Adresse: %s\n"
"Nähere Informationen: %s"

#: ../Pyblio/Resource.py:137
msgid "Starting application ..."
msgstr "Anwendung wird gestartet…"

#: ../Pyblio/Resource.py:140
msgid "No application to view resource"
msgstr "Keine Anwendung zur Ansicht der Ressource verfügbar"

#: ../Pyblio/Resource.py:141
#, python-format
msgid ""
"For mime type %s, no entry found in \n"
"configuration option resource/viewers.\n"
"Please consider adding one.\n"
"URL: %s"
msgstr ""
"Für den MIME-Typ %s wurde kein Eintrag in der\n"
"Konfigurationsoption Ressource/Betrachter gefunden.\n"
"Fügen Sie bitte einen Eintrag hinzu.\n"
"Adresse: %s"

#: ../Pyblio/Style/Parser.py:261
#, python-format
msgid "missing '%s' attribute"
msgstr "»%s«-Attribut fehlt"

#: ../Pyblio/Style/Parser.py:271
#, python-format
msgid "invalid opening tag: %s"
msgstr ""
"Ungültige Öffnungsmarkierung:\n"
"%s"

#: ../Pyblio/Style/Parser.py:281
#, python-format
msgid "invalid closing tag: %s"
msgstr "Ungültige Schlussmarkierung: %s"

#: ../pybliographer.py:43
#, python-format
msgid "This is %s, version %s"
msgstr "Willkommen zu %s, Version %s"

#: ../pybliographer.py:47 ../pybliographer.py:54
msgid "This is free software with ABSOLUTELY NO WARRANTY."
msgstr "Dies ist freie Software OHNE JEDE GARANTIE."

#: ../pybliographer.py:48
msgid "For details, type `warranty'."
msgstr "Für Details, wählen Sie »warranty«."

#: ../pybliographer.py:111 ../pybliographer.py:163
#, python-format
msgid "%s: error: can't open file `%s'"
msgstr "%s: Fehler: kann Datei »%s« nicht öffnen"

#: ../pybliographer.py:124
#, python-format
msgid "For help, run %s and type `help' at the prompt"
msgstr "Für Hilfe %s starten und »help« auf der Befehlszeile eingeben"

#: ../pybliographer.py:133
msgid ""
"Useful commands:\n"
"\thelp     to get some help\n"
"\tquit     to quit\n"
msgstr ""
"Nützliche Befehle:\n"
"\thelp     für Hilfe\n"
"\tquit     zum Beenden\n"

#. save history
#: ../pybliographic.desktop.in.h:1
msgid "Bibliography Manager"
msgstr "Bibliographieverwaltung"

#: ../pybliographic.desktop.in.h:2
msgid "Manage bibliographic databases"
msgstr "Bibliographische Datenbanken verwalten"

#: ../pybliographic.desktop.in.h:3
msgid "Pybliographic Bibliography Manager"
msgstr "Pybliographic Bibliographie-Verwaltung"

#: ../scripts/pybcheck.py:31
msgid "usage: pybcheck <file | directory>..."
msgstr "Aufruf: pybcheck <Datei | Ordner> …"

#: ../scripts/pybcheck.py:60
#, python-format
msgid "file `%s' is ok [%d entries]"
msgstr "Datei »%s« ist in Ordnung [%d Einträge]"

#: ../scripts/pybcompact.py:30
msgid "usage: pybcompact <latexfile> <bibtexfiles...>"
msgstr "Benutzung: pybcompact <latex-Datei> <bibtex-Dateien …>"

#: ../scripts/pybcompact.py:34
#, python-format
msgid "pybcompact: error: %s\n"
msgstr "pybcompact:-Fehler: %s\n"

#: ../scripts/pybcompact.py:99
msgid "no entry"
msgstr "kein Eintrag"

#: ../scripts/pybcompact.py:162 ../scripts/pybtext.py:240
#, python-format
msgid "can't find the following entries: %s"
msgstr "die folgenden Einträge können nicht gefunden werden: %s"

#: ../scripts/pybconvert.py:29
msgid "usage: pybconvert <source>..<target> <input> [output]"
msgstr "Benutzung: pybconvert <Quelle>..<Ziel> <Eingabe> [Ausgabe]"

#: ../scripts/pybconvert.py:38
msgid "pybconvert: error: bad conversion format"
msgstr "pybconvert: Fehler: Falsches Ausgabeformat"

#: ../scripts/pybformat.py:34
msgid ""
"usage: pybformat [options] <database...>\n"
"\n"
"    options:\n"
"      -o file, --output=file\t\tspecify an output filename\n"
"      -s style, --style=style\t\tspecify a bibliography style\n"
"      -f format, --format=format\tspecify an output format\n"
"      -H header, --header=header\tdefines a header file\n"
"      -F footer, --footer=footer\tdefines a footer file\n"
"      -l 'output', --list='output'\tlists the available output formats\n"
"      -h, --help\t\t\tshow this help message\n"
"      -v, --version\t\t\tshow this help message\n"
"    "
msgstr ""
"Aufruf: pybformat [Optionen] <Datenbank…>\n"
"\n"
"    Optionen:\n"
"      -o Datei, --output=Datei\t\tEinen Ausgabedateinamen angeben\n"
"      -s Stil, --style=Stil\t\tEinen Bibliographiestil angeben\n"
"      -f Format, --format=Format\tEin Ausgabeformat angeben\n"
"      -H Kopf, --header=Kopf\tEine Kopfdatei angeben\n"
"      -F Fuß, --footer=footer\tEine Fußdatei angeben\n"
"      -l 'output', --list='output'\tAlle verfügbaren Ausgabeformate "
"auflisten\n"
"      -h, --help\t\t\tDiese Hilfe anzeigen\n"
"      -v, --version\t\t\tVersionsinformationen anzeigen\n"
"    "

#: ../scripts/pybformat.py:49
#, python-format
msgid "pybformat: error: %s\n"
msgstr "pybformat: Fehler: %s\n"

#: ../scripts/pybformat.py:55
#, python-format
msgid "pybformat: warning: %s\n"
msgstr "pybformat: Warnung: %s\n"

#: ../scripts/pybformat.py:90 ../scripts/pybtext.py:252
#: ../scripts/pybtext.py:278 ../scripts/pybtext.py:321
#, python-format
msgid "can't open `%s': %s"
msgstr "»%s« konnte nicht geöffnet werden: %s"

#: ../scripts/pybformat.py:97
#, python-format
msgid "unknown list `%s'"
msgstr "Unbekannte Liste »%s«"

#: ../scripts/pybformat.py:100
#, python-format
msgid "pybformat: available values for `%s':"
msgstr "pybformat: Verfügbare Variablen für »%s«"

#: ../scripts/pybformat.py:104
#, python-format
msgid "empty value list `%s'"
msgstr "Leere Variablenliste »%s«"

#: ../scripts/pybformat.py:132
msgid "too few arguments"
msgstr "Zu wenige Argumente"

#: ../scripts/pybformat.py:139 ../scripts/pybtext.py:117
#, python-format
msgid "unknown output format `%s'"
msgstr "Unbekanntes Ausgabeformat »%s«"

#: ../scripts/pybformat.py:152 ../scripts/pybtex.py:86 ../scripts/pybtext.py:94
#, python-format
msgid "can't find style `%s'"
msgstr "Stil »%s« wurde nicht gefunden"

#: ../scripts/pybformat.py:155
#, python-format
msgid "pybformat: using style `%s', format `%s'\n"
msgstr "pybformat: Benutzung von Stil »%s«, Format »%s«\n"

#: ../scripts/pybformat.py:170
#, python-format
msgid "can't open header `%s': %s"
msgstr "Kopf »%s« konnte nicht geöffnet werden: %s"

#: ../scripts/pybformat.py:178
#, python-format
msgid "can't open database: %s"
msgstr "Datenbank konnte nicht geöffnet werden: %s"

#: ../scripts/pybformat.py:193
#, python-format
msgid "can't open footer `%s': %s"
msgstr "Fuß »%s« konnte nicht geöffnet werden: %s"

#: ../scripts/pybtex.py:34
msgid "usage: pybtex <latexfile> [bibtexfiles...]"
msgstr "Benutzung: pybtex <latex-Datei> [bibtex-Dateien…]"

#: ../scripts/pybtex.py:38
#, python-format
msgid "pybtex: error: %s\n"
msgstr "pybtex: Fehler: %s\n"

#. warn the user that some entries were not found
#: ../scripts/pybtex.py:59
msgid "pybtex: warning: the following keys were not resolved"
msgstr "pybtex: Warnung: die folgenden Schlüssel wurden nicht aufgelöst"

#. If the LaTeX document declares no style...
#: ../scripts/pybtex.py:64
msgid "no style defined"
msgstr "kein Stil definiert"

#: ../scripts/pybtext.py:41
msgid "usage: pybtext [-o outputfile] [-s style] <textfile> <bibfiles...>"
msgstr "Aufruf: pybtext [-o Ausgabedatei] [-s Stil] <Textdatei> <Bib-Dateien…>"

#: ../scripts/pybtext.py:45
#, python-format
msgid "pybtext: error: %s\n"
msgstr "pybtext: Fehler: %s\n"

#: ../scripts/pybtext.py:50
#, python-format
msgid "pybtext: warning: %s\n"
msgstr "pybtext: Warnung: %s\n"

#: ../scripts/pybtext.py:124
#, python-format
msgid "File already exists: `%s'"
msgstr "Datei ist bereits vorhanden: »%s«"

#: ../scripts/pybtext.py:127
#, python-format
msgid "A file with the same name already exists: `%s'"
msgstr "Eine Datei mit gleichem Namen ist bereits vorhanden: »%s«"

#: ../scripts/pybtext.py:178
msgid "no citation found"
msgstr "kein Zitat gefunden"

#: ../scripts/pybtext.py:193
#, python-format
msgid "pybtext: using style `%s', format `%s'\n"
msgstr "pybtext: Benutzung von Stil »%s«, Format »%s«\n"

#: ../scripts/pybtext.py:316
#, python-format
msgid "can't create `%s'"
msgstr "»%s« konnte nicht erstellt werden"

#: ../scripts/pybtext.py:342
#, python-format
msgid "can't remove `%s'"
msgstr "»%s« konnte nicht gelöscht werden"

#: ../scripts/pybtext.py:344
msgid "Done"
msgstr "Fertig"

#~ msgid "Failure for item %s"
#~ msgstr "Fehler für das Objekt %s"

#~ msgid ""
#~ "No viewer found in Config resource/viewers\n"
#~ "URL: %s"
#~ msgstr ""
#~ "Kein Betrachter in der Konfiguration Ressourcen/Betrachter gefunden\n"
#~ "Adresse: %s"

#~ msgid "Add..."
#~ msgstr "Hinzufügen…"

#~ msgid "Edit..."
#~ msgstr "Bearbeiten…"

#~ msgid "Delete..."
#~ msgstr "Löschen…"
